.github/actions/dotnet-tests-prepare-outputs/action.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Prepare dotnet tests outputs
+
+#
+# This action is called from the 'dotnet-solution-ci-build.yml' file.
+#
+# Prepared outputs will be zipped and uploaded as the 'dotnet-tests' artifact.
+#
+
+inputs:
+  OUTPUT_PATH:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    # To ensure code coverage tooling is available in bin folder, we use publish on test assemblies
+    # See https://github.com/coverlet-coverage/coverlet/issues/521#issuecomment-522429394
+    - name: Publish WebApi
+      shell: bash
+      run: |
+        dotnet publish \
+          '.\apps\dh\api-dh\source\DataHub.WebApi.Tests\DataHub.WebApi.Tests.csproj' \
+          --no-build \
+          --no-restore \
+          --configuration Release \
+          --output '.\apps\dh\api-dh\source\DataHub.WebApi.Tests\bin\Release\net6.0'
+
+    # To ensure code coverage tooling is available in bin folder, we use publish on test assemblies
+    # See https://github.com/coverlet-coverage/coverlet/issues/521#issuecomment-522429394
+    - name: Publish Tests
```

Comment:
This step should not be necessary. If it is I would like to understand why? We should only have to publish test projects here.